README.md

```diff
@@ -25,7 +25,7 @@ Minimum Litestar Implementation.
 
 `$ poetry install`
 
-`$ poetry run uvicorn main:app --reload`
+`$ litestar run --reload`
```

Comment:
Will this work outside of the venv?

Reply:
Good point, what if I change the first command to `$ poetry shell && poetry install`?

Reply:
That would work, but I'd swap the order to `poetry install && poetry shell`

Reply:
Sure - I'll do it that way, what is the difference?

Reply:
I think the venv and project need to be installed before you shell in, but im not really sure how the poetry magic works that deep

Reply:
but also maybe you could do `poetry run litestar run --reload`? idk

Reply:
> I think the venv and project need to be installed before you shell in

That's a really good point.

Every time before I install I always run `poetry env use 3.x.x`, which automatically drops me into a poetry shell.. so I've only ever installed after entering the shell and not found it to be an issue.

What I have noticed is flakey-ness around whether I actually end up in a shell at the terminal or not.

Without that `poetry env use ...` statement `poetry install` will always use my system python, but then sometimes it doesn't enter the shell properly (i.e.,, I don't get the `(name-of-venv-in-brackets)` at the start of the prompt. When that happens, if I run `poetry activate` again, it already thinks I'm in the shell, but if I run `deactivate ` it uses the pyenv python version, not the system python version - so I can't exit out of the poetry shell. Something to do with the poetry sub shell maybe... anyway probably digressing.

I think I'm just going to split the double command line into 2.

Reply:
The reason I don't want to use `poetry run litestar run` is that I think it shows off the CLI better if the command is simply `litestar run`.